bundles/org.openhab.binding.electroluxair/README.md

```diff
@@ -26,15 +24,15 @@ Only the bridge require manual configuration. The Electrolux Pure A9 thing can b
 
 | Parameter | Description                                                  | Type   | Default  | Required |
 |-----------|--------------------------------------------------------------|--------|----------|----------|
-| username  | The username used to connect to the Electrolux Wellbeing app | String | NA       | yes      |
-| password  | The password used to connect to the Electrolux Wellbeing app | String | NA       | yes      |
+| username  | The username used to connect to the Electrolux app | String | NA       | yes      |
+| password  | The password used to connect to the Electrolux app | String | NA       | yes      |
```

Comment:
Can you restore for table formatting here?

Reply:
Fixed!